NOTES.md

```diff
@@ -0,0 +1,36 @@
+# Things Paul Came Across
+
+## General Python housekeeping I could do later
+
+- *Proper package*. Make imports more normal by changing `t-html` to something legally importable.
+- *pyproject.toml*. Put dependencies in there, then put everything under `src/thtml`. This would mean moving the
+  PyScript demo code up to root.
+- *Type hints*. Spread more of these around, for those that use IDEs that can autocomplete and squiggle.
```

Comment:
currently what people should use is just `render`, `html` and `svg` ... latter two are the same and render is simple enough but I would like to not use modules not available in MicroPython, considering MicroPython supports basic type hint already so any help in there would be appreciated